packages/edit-site/src/components/global-styles/screen-block-list.js

```diff
@@ -156,17 +177,7 @@ function ScreenBlockList() {
 				label={ __( 'Search for blocks' ) }
 				placeholder={ __( 'Search' ) }
 			/>
-			<div
-				ref={ blockTypesListRef }
-				className="edit-site-block-types-item-list"
-			>
-				{ filteredBlockTypes.map( ( block ) => (
-					<BlockMenuItem
-						block={ block }
-						key={ 'menu-itemblock-' + block.name }
-					/>
-				) ) }
-			</div>
+			<MemoizedBlockList filterValue={ defferedFilterValue } />
```

Comment:
Typo:
```suggestion
			<MemoizedBlockList filterValue={ deferredFilterValue } />
```

Reply:
Fixed in [efc9f3a](https://github.com/WordPress/gutenberg/pull/56139/commits/efc9f3a29f97fbe66999bb589ce0d3f52f99fb7f).

Thanks for spotting the typo. One day, I'll learn how to type "deferred" and "unnecessary" 😅